pkg/operator/operator_test.go

```diff
@@ -74,7 +75,8 @@ func newFakeOperator(kubeObjects []runtime.Object, osObjects []runtime.Object, s
 	}
 
 	configSharedInformer.Start(stopCh)
-	kubeNamespacedSharedInformer.Start(stopCh)
+	kubeInformerstopCh := make(<-chan struct{})
```

Comment:
Why do we need this separate channel? Shouldn't the following line use `stopCh`?

Reply:
Agree, please undo the change.